tools/clang/test/HLSLFileCheck/dxil/debug/value_cache/load.hlsl

```diff
@@ -0,0 +1,28 @@
+// RUN: %dxc -E main -T ps_6_6 %s -Od | FileCheck %s
+
+Texture2D tex0 : register(t0);
+Texture2D tex1 : register(t42);
+
+const static float2 my_offsets[] = {
+  float2(1,2),
+  float2(3,4),
+  float2(5,6),
+  float2(7,8),
+};
+
+[RootSignature("DescriptorTable(SRV(t0), SRV(t42)), DescriptorTable(Sampler(s0))")]
+float4 main(uint2 uv : TEXCOORD) : SV_Target {
+  // CHECK: %[[handle:.+]] = call %dx.types.Handle @dx.op.createHandle(i32 57, i8 0, i32 1, i32 42
+  int x = 0;
+  int y = 0;
+
+  float2 val = my_offsets[x+y+1];
+
+  Texture2D tex = tex0;
```

Comment:
If we can make tex0 a dynamically indexed resource array, that would help exersize the case where you have to drill through the GEP.